trustd/src/main.rs

```diff
@@ -1,19 +1,17 @@
 use clap::Parser;
-use postgresql_embedded::PostgreSQL;
 use std::env;
 use std::process::{ExitCode, Termination};
-use tokio::task::JoinSet;
-use trustify_auth::auth::AuthConfigArguments;
-use trustify_auth::swagger_ui::SwaggerUiOidcConfig;
-use trustify_common::config::{Database, DbStrategy, StorageConfig};
-use trustify_common::db::CreationMode;
-use trustify_infrastructure::app::http::HttpServerConfig;
-use trustify_infrastructure::endpoint::Trustify;
-use trustify_infrastructure::InfrastructureConfig;
+
+mod db;
 
 #[allow(clippy::large_enum_variant)]
 #[derive(clap::Subcommand, Debug)]
-pub enum Command {}
+pub enum Command {
+    /// Run the API server
+    Api(trustify_server::Run),
+    /// Manage the database
+    Db(db::Run),
```

Comment:
I think we'd also want the `Importer` loop command?

Reply:
Possibly. Out of scope for this PR, though. I just want to make PM mode make sense.